python/django/cockroach_example/views.py

```diff
@@ -21,13 +51,17 @@ def get(self, request, id=None, *args, **kwargs):
             customers = list(Customers.objects.filter(id=id).values())
         return JsonResponse(customers, safe=False)
 
+    @retry_on_exception(3)
+    @atomic
```

Comment:
I think the execution of view needs an atomic, not the view itself. Or does this first wrap the function with retry execption, and then atomic, so its doing the same thing?

Reply:
It wraps the function with atomic, and then wraps the result in retry.